auditbeat/module/file_integrity/metricset_test.go

```diff
@@ -487,12 +437,11 @@ func (e expectedEvent) validate(t *testing.T, ms *MetricSet) {
 type expectedEvents []expectedEvent
 
 func (e expectedEvents) validate(t *testing.T) {
-	store, err := ioutil.TempFile("", "bucket")
+	store, err := os.CreateTemp(t.TempDir(), "bucket")
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer store.Close()
```

Comment:
Not a big deal, but I'd leave the deferred close. The file's finalizer will do a close anyway, but it is good practice to pretend the language doesn't have them. (also below)